webapp/src/pages/ChatroomPage/components/ChatroomMessages/ChatroomMessages.tsx

```diff
@@ -96,6 +96,10 @@ function ChatroomMessages({ from }: ChatroomMessagesProps) {
     };
   }, [from]);
 
+  if (blockedUsers === null) {
+    return null;
+  }
+
```

Comment:
This change fixes a bug where messages from the blocked users appear on the first render because the `blocks` variable is empty until the fetch completes.